test/unit/accounts/find-all-test.js

```diff
@@ -0,0 +1,71 @@
+var simple = require('simple-mock')
+var test = require('tap').test
+
+var findAllAccount = require('../../../lib/accounts/find-all')
+
+test('findAllAccount', function (group) {
+  group.test('with `options` in arguments', function (t) {
+    t.plan(1)
+
+    var state = {
+      db: {
+        get: simple.stub().resolveWith({
+          _id: 'org.couchdb.user:foo',
+          _rev: '1-234',
+          name: 'foo',
+          roles: ['id:user123'],
+          profile: {
+            name: 'bar'
+          }
+        }),
+        put: simple.stub().resolveWith({})
```

Comment:
as these are unit tests, we need to mock all outside state and logic. Here you are mocking the PouchDB instance `state.db` and its `.get()` and `.put()` methods. The error that you currently get is `state.db.allDocs is not a function`. Which tells you that `findAllAccount()` calls `state.db.alldocs()` ([here](https://github.com/hoodiehq/hoodie-account-server-api/blob/2a5785502ffe6301e137dbc66e76757ee8cf8415/lib/accounts/find-all.js#L6)), so you need to mock its behaviour